### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/markdown/container.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#674] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/markdown/container.html

主题可以为你添加提示、注释、信息、注意、警告和详情自定义容器的支持。

https://theme-hope.vuejs.press/zh/guide/markdown/container.html

**Comment by vseasky** [comment]
如果可以支持obsidian 的标签就好了，这样Obsidian的一些笔记转移到Vuepress-hope就很方便了。这里有一个样式参考：https://www.liuwei.vin/670.html ，这个需求可能也只是我个人的小众需求。

obsidian提示标签的格式是
>[!important]-
>Lorem ipsum dolor sit amet

**Reply by Mister-Hope** [reply]
我们现在支持了 GFM 的 Alert 语法，应该和你说的一致或者很相似
